libs/components/src/lib/menu-item/menu-item.ts

```diff
@@ -16,6 +16,14 @@ export class MenuItem extends FastMenuItem {
 	 * HTML Attribute: text
 	 */
 	@attr text?: string;
+	/**
+	 * Indicates the menu item's secondary text.
+	 *
+	 * @public
+	 * @remarks
+	 * HTML Attribute: text
+	 */
+	@attr({ attribute: 'secondary-text' }) secondaryText?: string;
```

Comment:
should this be ordered by category first?
```suggestion
	@attr({ attribute: 'text-secondary' }) textSecondary?: string;
```